system-addon/test/unit/activity-stream-prerender.test.jsx

```diff
@@ -0,0 +1,60 @@
+const prerender = require("content-src/activity-stream-prerender");
+const {prerenderStore} = prerender;
+const {PrerenderData} = require("common/PrerenderData.jsm");
+
+describe("prerenderStore", () => {
+  it("should create a store", () => {
+    const store = prerenderStore();
+    assert.isFunction(store.getState);
+  });
+  it("should start uninitialized", () => {
+    const store = prerenderStore();
+    const state = store.getState();
```

Comment:
In all of these tests, calling getState wants to be part of the verification block, not the execution block.